src/components/button-toggle/button-toggle.pw.tsx

```diff
@@ -227,8 +227,9 @@ test.describe("Prop tests", () => {
   });
 
   ([
-    [SIZE.SMALL, 32],
-    [SIZE.LARGE, 48],
+    [SIZE.SMALL, 24],
+    [SIZE.MEDIUM, 32],
```

Comment:
**question:** is it right that these `MEDIUM` tests have snuck back in since the QA review?

Reply:
The Medium button size was removed in error. It was just the Medium buttonIconSize that needed to be removed :)